pom.xml

```diff
@@ -62,20 +62,20 @@
     <dependency>
       <groupId>org.jenkins-ci.modules</groupId>
       <artifactId>ssh-cli-auth</artifactId>
-      <version>1.1</version>
+      <version>1.4</version>
```

Comment:
The only potential issue is that backporting of this change (e.g. SSHD Module 1.12 or above) will also require bumping the ssh-cli-auth module. I do not see any issue with that in this case. The modules have to evolve anyway